zinc/src/sbt-test/macros/macro-type-change-3/pending

```diff
@@ -0,0 +1,3 @@
+> app/run false
```

Comment:
Plan: defer this part. This is the same problem in https://github.com/sbt/zinc/pull/1282#issuecomment-1832399563